.circleci/config.yml

```diff
@@ -168,6 +253,19 @@ workflows:
       - run-tests:
           run-unit-tests: false
           context: vscode-test
+      - release:
```

Comment:
Only runs when updates are pushed to `master` and if all the tests are successful